task/buildah-oci-ta/0.3/buildah-oci-ta.yaml

```diff
@@ -725,3 +754,26 @@ spec:
         requests:
           cpu: 100m
           memory: 256Mi
+  sidecars:
+    - name: indy-generic-proxy
+      image: quay.io/factory2/indy-generic-proxy-service:latest-prod
+      volumeMounts:
+        - name: indy-generic-proxy-secrets
+          readOnly: true
+          mountPath: /mnt/secrets-generic-proxy
+        - name: indy-generic-proxy-config
+          readOnly: true
+          mountPath: /deployment/config
+      computeResources:
+        limits:
+          cpu: 1
+          memory: 2Gi
+        requests:
+          cpu: 200m
+          memory: 512Mi
+      script: |
+        if [ "$(params.ENABLE_INDY_PROXY)" == "true" ]; then
+          export QUARKUS_OIDC_CLIENT_CLIENT_ID="$(params.INDY_PROXY_CLIENT_ID)"
+          export QUARKUS_OIDC_CLIENT_CREDENTIALS_SECRET="$(params.INDY_PROXY_CLIENT_CREDENTIAL)"
+          /deployment/start-service.sh
+        fi
```

Comment:
If we are to implement a sidecar like this, is there any reason why it needs to be specific to Quarkus/INDY? Would this have value as a generalization?

Reply:
Both `QUARKUS_OIDC_CLIENT_CLIENT_ID ` and `QUARKUS_OIDC_CLIENT_CREDENTIALS_SECRET ` are supported in quarkus that we can declare the variable as env, which can be declared in the config file as following: 

```
 quarkus
    oidc-client:
       auth-server-url: ""
       client-id: 
       credentials:
           secret: 
```

Reply:
QUARKUS_* are internal, I believe it doesn't really matter how they are called.
*_INDY_* should these be renamed to more generic name like HTTP_PROXY_* ?